packages/shared/src/write-archive/archive-file.ts

```diff
@@ -101,15 +101,13 @@ export class ArchiveFile {
   }
 
   private addExtension(pathname: string) {
-    if ('error' in this.response || !this.response.contentType) return pathname;
+    if ('error' in this.response) return pathname;
 
     // Add an extension if needed
     let nameWithExtension = pathname;
     if (!path.extname(nameWithExtension)) {
-      const fileExtension = mime.getExtension(this.response.contentType);
-      if (fileExtension) {
-        nameWithExtension = `${pathname}.${fileExtension}`;
-      }
+      const fileExtension = mime.getExtension(this.response.contentType) || 'tmp';
```

Comment:
@tevanoff is it safe to call `mime.getExtension` without a content type, or would it be better to only call it if there is a content type? Just want to make sure it wouldn't throw an error or something, as I didn't see mention of whether the content type is a required parameter in the [docs](https://www.npmjs.com/package/mime).

Reply:
It's very rare and unlikely to not have a Context Type on a response, but I guess it's not impossible here.

Reply:
Yeah it's more likely that the content type doesn't match to a file extension, but in either case the mime library will return `null`, so all's good. The unit test covers this.